File: 프로그래머스/lv1/42576. 완주하지 못한 선수/완주하지 못한 선수.cpp
#include <bits/stdc++.h>
using namespace std;

string solution(vector<string> participant, vector<string> completion) {
    map<string, int> m;

	for(string s : participant) m[s]++;
	for(string s : completion) m[s]--;

	for(auto x : m)
		if(x.second != 0)
			return x.first;		
}

File: 프로그래머스/1/42576. 완주하지 못한 선수/완주하지 못한 선수.py
from collections import defaultdict

def solution(participant, completion):
    dict = defaultdict(int)
    answer = 0

    for p in participant:
        dict[p] += 1
    for c in completion:
        dict[c] += 1

    for k, v in dict.items():
        if v % 2 != 0:
            return k